.changes/next-release/Bug Fix-2564ea73-0e4a-4404-ab79-14710c867bea.json

```diff
@@ -0,0 +1,4 @@
+{
+	"type": "Bug Fix",
+	"description": "Authenticating through the browser now requires users to manually enter a user verification code. Previously this was done automatically."
```

Comment:
should we mention "AWS Builder ID" here

Reply:
It applies to all SSO logins but I suppose we could add specific examples somewhere